networks/suzuka/suzuka-client/src/tests/mod.rs

```diff
@@ -182,76 +182,6 @@ async fn test_example_interaction() -> Result<(), anyhow::Error> {
 			.context("Failed to get Bob's account balance the second time")?
 	);
 
-	// malformed sequence number
-	println!("\n=== Malformed Sequence Number ===");
```

Comment:
If this test code is still useful, it could be moved to its own test case that is marked with `#[ignore]` if it has to be disabled at the moment.

Reply:
Without the `ValidationVM::validate` call in the mempool, it will not pass.

Reply:
We still have that code, albeit commented out. The ignored test would remind us to add it in.